server/routes/bookedtoday/unexpectedArrivals/arrivals/multipleExistingRecordsFoundController.ts

```diff
@@ -0,0 +1,12 @@
+import type { RequestHandler } from 'express'
+import type { ExpectedArrivalsService } from '../../../../services'
+
+export default class MultipleExistingRecordsFoundController {
+  public constructor(private readonly expectedArrivalsService: ExpectedArrivalsService) {}
+
```

Comment:
I think maybe best not to pass services down to controllers until we need them there

Reply:
Also maybe the arrival subfolder within unexpectedArrivals is unnecessary at this stage?

Reply:
agreed

Reply:
agreed